test/90_1_prm/test_appendix_g_prm.rb

```diff
@@ -1186,6 +1186,90 @@ def check_sat_ctrl(prototypes_base)
     end
   end
 
+  # Check if split of zones to PSZ from multizone baselines is working correctly
+
+  def check_psz_split_from_mz(prototypes_base)
+
+    prototypes_base.each do |prototype, model|
+      building_type, template, climate_zone, mod = prototype
+
+      # Concatenate modifier functions and arguments
+      mod_str = mod.flatten.join('_') unless mod.empty?
+
+      run_id = "#{building_type}_#{template}_#{climate_zone}_#{mod_str}"
+      @bldg_type_alt_now = @bldg_type_alt[prototype]
+
+      if building_type == 'MediumOffice' && mod_str == 'remove_transformer_change_zone_epd_Perimeter_bot_ZN_1 ZN_70'
+        # This mod should isolate Perimeter_bot_ZN_1 ZN to PSZ
+        # Fan schedule for the PSZ should be same as the MZ system fan schedule (92 hrs/wk)
+        # MZ system will have the zone Core_bottom ZN on it
+        # Review all air loops and check zones and fan schedules 
+        num_zones_target = 0
+        num_zones_mz = 0
+        fan_hrs_per_week_target = 0
+        fan_hrs_per_week_mz = 0
+        model.getAirLoopHVACs.each do |air_loop|
+          air_loop.thermalZones.each do |zone|
+            zone_name = zone.name.get
+            if zone.name.get == 'Perimeter_bot_ZN_1 ZN'
+              # Get fan hours and num zones
+              num_zones_target = air_loop.thermalZones.size
+              fan_hrs_per_week_target = get_fan_hours_per_week(model, air_loop)
+            elsif zone.name.get == 'Core_bottom ZN'
+              num_zones_mz = air_loop.thermalZones.size
+              fan_hrs_per_week_mz = get_fan_hours_per_week(model, air_loop)
+            end
+          end
+        end
+
+        assert((num_zones_target == 1 && num_zones_mz > 1), "Split PSZ from MZ system fails for high internal gain zone.")
```

Comment:
Looks great! Did you not want to include a test on the fan schedule hours? I see that you're retrieving them but not doing anything with them.